### PR TITLE
fix: sort helm list by last deployed

### PR DESCRIPTION
**Pull request by rish2320**
# Description
1-Helm Apps List-Last deployed at' sorting missing
2-Show sort arrow on all sortable columns by default

Fixes #[3906](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/OSS%20adoption/Stories/?workitem=3906)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Tested all the cases manually.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/list-new/HelmAppList.tsx [hunk 1]
```diff
@@ -393,7 +398,14 @@ export default function HelmAppList({
                     <span className="app-list__cell-header">{APP_LIST_HEADERS.Namespace}</span>
                 </div>
                 <div className="app-list__cell app-list__cell--time">
-                    <span className="app-list__cell-header">{APP_LIST_HEADERS.LastDeployedAt}</span>
+                    <button className="app-list__cell-header flex" onClick={sortByLastDeployed}>
```
Comment (vivek-devtron) [inline src/components/app/list-new/HelmAppList.tsx:401]: Why this button, can't it be done via some span?

File: src/components/ClusterNodes/ClusterDetails.tsx [hunk 2]
```diff
@@ -537,12 +536,26 @@ export default function ClusterDetails({ imageList, isSuperAdmin, namespaceList,
...
+        }
+    }
+
     const renderNodeListHeader = (column: ColumnMetadataType): JSX.Element => {
         const nodeColumnClassName = fixedNodeNameColumn ? 'bcn-0 dc__position-sticky  sticky-column dc__border-right' : ''
         return (
             <div
                 className={`h-36 list-title dc__inline-block mr-16 pt-8 pb-8 ${
-                    column.label === 'Node' ? `${nodeColumnClassName} w-280 pl-20` : 'w-100px'
+                    column.label === 'Node' ? `${nodeColumnClassName} w-280 pl-20` : 'w-150'
```
Comment (vivek-devtron) [inline src/components/ClusterNodes/ClusterDetails.tsx:558]: Please verify and fix the scroll

File: src/components/app/list-new/HelmAppList.tsx (hunk 1 above)
Comment (rish2320) [reply src/components/app/list-new/HelmAppList.tsx:401]: done

File: src/components/ClusterNodes/ClusterDetails.tsx (hunk 2 above)
Comment (rish2320) [reply src/components/ClusterNodes/ClusterDetails.tsx]: doing w-120px works fine

Comment (rish2320) [reply src/components/ClusterNodes/ClusterDetails.tsx]: done

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**